ols/src/llms/llm_loader.py

```diff
@@ -302,12 +301,12 @@ def _watson_llm_instance(self) -> LLM:
             "url": (
                 self.llm_params.get("url")
                 if self.llm_params.get("url") is not None
-                else os.environ.get("WATSON_API_URL", None)
+                else self.provider_config.url
             ),
             "apikey": (
                 self.llm_params.get("apikey")
                 if self.llm_params.get("apikey") is not None
```

Comment:
same here, we should not try to get this value from llm_params, only from the config.